Scripts/fastlane/Fastfile

```diff
@@ -186,6 +185,23 @@ import "./ScreenshotFastfile"
     close_milestone(repository:GHHELPER_REPO, milestone: version)
   end
 
+  #####################################################################################
+  # new_hotfix_release
```

Comment:
This should probably say `finalize_hotfix_release`, I think?

Reply:
Thanks! Fixed!